Update owasp-crs-response.mdx (#1532)
ngrok/ngrok-docs: docs/traffic-policy/actions/owasp-crs-response.mdx

import { YouTubeEmbed } from "@components/youtube-embed";

> In this video, I'll walk you through ngrok's OWASP CRS request and response actions in Traffic Policy, which let you add web application firewall protection directly to your ngrok endpoints without changing your existing infrastructure.

Reviewer: style preference: avoid "I"

```suggestion
The video below walks you through ngrok's OWASP CRS request and response actions in Traffic Policy, which let you add web application firewall protection directly to your ngrok endpoints without changing your existing infrastructure.
```
